### ISSUE TITLE
Cannot upload file for episode

### ISSUE THREAD

**Issue by tedkeylon** [PodcastGenerator/PodcastGenerator#385]
**System Information**
Podcast Generator Version: 3.1
Webserver: OLM.net
PHP Version: ?

**Podcast Generator Config**

```
INSERT THE **CENSORED** CONFIG HERE
``` 
https://thespectra.net/podcast/pg-cron.php?key=dznp9yrp
**Description of your problem:**
Cannot upload files for episodes

**Comment by tedkeylon** [comment]
I thought it was because the file was too big but I made it 25 minutes and did not get a prompt that time but still none of the episodes are showing

**Comment by samisaacscreative** [comment]
Hmmmm, I have had a look at the site and nothing visible from my side. Your web hosting provider my not support certain PHP Variables, which may stop the code uploading the files. Do you have FTP access? If so, are the mp3's in the media folder on the server?

**Comment by tedkeylon** [comment]
The media folder in podcast is empty... 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
I found uploading via FTP was the only way for me to upload episodes.

**Comment by tedkeylon** [comment]
Aha! Thank you, once I uploaded directly into the media folder, it worked!

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
> 
> Aha! Thank you, once I uploaded directly into the media folder, it worked! Ted Keylon (217) 691-1301 tedkeylon@gmail.com eted@icloud.com
> […](#)
> On Jan 4, 2021, at 1:26 PM, Rob Howdle ***@***.***> wrote: ﻿ I found uploading via FTP was the only way for me to upload episodes. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Glad it worked!

**Comment by tedkeylon** [comment]
Now a new problem, the feed.xml says it “This file does not appear to have any style information associated with it.”

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
> 
> Now a new problem, the feed.xml says it “This file does not appear to have any style information associated with it.” Ted Keylon (217) 691-1301 tedkeylon@gmail.com eted@icloud.com
> […](#)
> On Jan 5, 2021, at 4:38 PM, Rob Howdle ***@***.***> wrote: ﻿ Aha! Thank you, once I uploaded directly into the media folder, it worked! Ted Keylon (217) 691-1301 ***@***.*** ***@***.*** … On Jan 4, 2021, at 1:26 PM, Rob Howdle @.***> wrote: ﻿ I found uploading via FTP was the only way for me to upload episodes. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe. Glad it worked! — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Mine says that also. It does not affect anything in terms of the way it works.

**Comment by tedkeylon** [comment]
Well why is there as button there that says rss but when you click it it tells everyone something is not set up correctly? 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
Also the iTunes button does not seem to work on a phone.... I mean isn’t the point of a podcast so users can subscribe to the feed?? And not have to necessarily type in the url every time?

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
I don't know, you're asking various questions unrelated to the original post. It'd be better to open a new issue. The feed work by getting submitted to things such as apple providing you have set them up correctly. You must manually enter your feed URL into Apple. Once it's been accepted and updated, the user would then go into Apple Podcasts and be able to find your podcast that way. 

If you have set it up correctly the user should never have to type any url feed in. The point of this program is that it is a self hosted podcast generating platform. That means you upload your episode to it and you can create your own front end website if you wanted, stick with the default things if you want but then you would submit your feeds to the likes of Apple, Google and Spotify etc. If somebody wants to listen through iTunes they do that through itunes. No URLs involved, if they're typing in URLs, they are not listening through itunes.

**Comment by tedkeylon** [comment]
But don’t they need the feed to listen to it?

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
Of course they do. The feed is generated automatically when you upload an episode. Upload an episode, if you're uploading via FTP, make sure you go into the FTP section to index the episode, then edit the details as you need to and click save. That will automatically update your feed. If you have set things like Apple Podcasts up correctly, Apple will automatically refresh the feed to pull down the latest information.

**Comment by tedkeylon** [comment]
I mean, I tried to search by the name but nothing comes up... and my friend that tried to use the site just complained about that xml page coming up because the button is front and center and it’s the first thing anyone tries to do as of then they get a page of code that says it has no style ...

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
The episodes are at the bottom no problem but those buttons are at the top and they lead to confusion for people 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
I used a custom front end design when I did this for a friend of mine and his podcast. You can just hide the buttons or at least the RSS one if you don't need it.

**Comment by RobHowdle** [comment]
Admin>Theme & Aspect > Change Buttons allows you to change the URL of the buttons if you need to, add new buttons or remove them.

**Comment by tedkeylon** [comment]
Well they both are problematic but at least the iTunes one looks like it’s apples fault and not mine.. yeah I’ll put a button to a different place ...

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
Check the URL. Depending on how recently you got your show on Apple Podcasts it might not have propagated yet.

**Comment by tedkeylon** [comment]
Ah on both counts... thank you...

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
That second button keeps inserting the iTunes protocol even if I delete and save 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
So create a new button instead of reusing.

**Comment by tedkeylon** [comment]
Like I said, it keep putting the protocol back even if I delete and save

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
So I guess I just have to leave that one as iTunes but shit I’m done so that will have to do...

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
I'm not sure what the issue would be on that one. I just created a test button that linked to google, saved it and it worked perfectly fine. Can't help on that I don't think I'm afraid.

**Comment by tedkeylon** [comment]
It’s the second button that forces the iTunes protocol so I just changed it back and left it pointing to iTunes since it wants to so bad. The first button changed just fine but it never had a protocol in the field anyway. Plus the real problem was everyone complaining that they got a bunch of code when they clicked on rss button because they thought that was how they subscribed . At least now it just goes to my Facebook page for the podcast.... 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by Go-rom** [comment]
@tedkeylon is your issue solved ? It it is, please close this issue
If not, please close this issue and create a new one (help wanted) because you're totally of the original topic and it's not helping tickets managing. Thank you

**Comment by tedkeylon** [comment]
How do I get there again?



Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by Go-rom** [comment]
I'm sorry, could you rephrase your question please ?

**Comment by tedkeylon** [comment]
How do I get to where I can close the issue as you say? There should be a link in your email but I don’t see it so I can’t get there to close it because I don’t remember your url (uniform resource locator).

Ted

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
You opened the issue on github so go to github and check your notifications. At the bottom where you would respond click close issue. Or go to github and search for Podcast Generator.

**Comment by tedkeylon** [comment]
Well, that was mildly helpful I asked for a link. If it’s so important then there should be a link in these emails because that’s how I talked to people on the board is through email so the links should be here to easily return to close out the thread. Besides I’m annoyed about all the malarkey about what’s on subject. Don’t punish people for trying to use the software. This is more important to you than me so link or no close (I’m forcing you to improve your policies a simple footer with the way back is all it takes).

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
Not only that but I don’t even remember my login details so

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by RobHowdle** [comment]
You cannot open one issue and then discuss countless other things because it makes trying to solve the issue ten times harder then having to go through your million and one posts to find what you are actually having a problem with. It would certainly be easier if links were in emails but that's not the creators of this repos problem, it's Githubs issue, take it up with them. I'm fairly certain that you'd have had to come onto github to open the original issue but whatever. 
https://github.com/PodcastGenerator/PodcastGenerator/issues/385

**Comment by RobHowdle** [comment]
[https://github.com/PodcastGenerator/PodcastGenerator/issues/385](url)

**Comment by tedkeylon** [comment]
Thank you. 

Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by tedkeylon** [comment]
Well yeah I don’t remember my login details so you close it. The whole thing was a nightmare anyway, rude, illogical bureaucratic nonsense. I don’t have time for it. 


Ted Keylon
(217) 691-1301
tedkeylon@gmail.com
eted@icloud.com

**Comment by emilengler** [comment]
No problem! I (the maintainer of the project) will close it for you! Thanks for using Podcast Generator!
